src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/SectionsHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.github.onsdigital.babbage.template.handlebars.helpers;
+
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Options;
+import com.github.onsdigital.babbage.template.handlebars.helpers.base.BabbageHandlebarsHelper;
+import com.github.onsdigital.babbage.template.handlebars.helpers.markdown.CustomMarkdownHelper;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by angela on 04/06/21.
+ */
+public enum SectionsHelper implements BabbageHandlebarsHelper<Object> {
+
+    totalWordCount {
+        @Override
+        public CharSequence apply(Object context, Options options) throws IOException {
+            if (options.isFalsy(context) || ((List<Object>) context).isEmpty()) {
+                return null;
+            }
+
+            CustomMarkdownHelper mdHelper = new CustomMarkdownHelper();
+            Integer runningWordCount = 0;
+            for (Object section : (List<Object>) context) {
+                Map<String, String> sectionText = (Map<String, String>) section;
+
+                String md = sectionText.get("markdown");
```

Comment:
Wondering if we could extract two methods from these `wordCount` blocks to have something like:

```java
String mdWordCount = calculateWordCountFromMarkdown(md)
String titleWordCount = calculateWordCountFromSection(title)
```

Or something to that effect? Main reason I'm thinking this is that it could make the public method easier to read if we're able to hide away some of the implementation and describe it in a clear method name

Another question: could these variables be `int` instead? Let me know what you think

Reply:
refactored into private methods and converted variables to ints.